### PR TITLE
Make addable as dependency

### PR DESCRIPTION
**Pull request by bhuminson**
We want to be able to add the footer here as a dependency in nextjs-community and ifixit in the future. It's not ideal to keep the tarball around, and publishing it as an npm package will add some extra work to maintain so we plan to add this git repo as a dependency in the other repos' `package.lock`. However there's a couple things preventing that from happening:

1. Export target doesn't start with `./` so an error is thrown at next.js build time
2. The source isn't getting bundled up when we install it as a dependency because there is no `prepare` in `package.lock`

This pull addresses those two issues.

CR should be straight forward

QA
--
I can handle this one. I'm gonna try adding it in nextjs-community with these changes.

Connects https://github.com/ifixit/ifixit/issues/39183

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,14 +9,14 @@
    "license": "UNLICENSED",
    "main": "dist/footer.js",
    "typings": "dist/index.d.ts",
-   "exports": "dist/footer.modern.js",
+   "exports": "./dist/footer.modern.js",
    "module": "dist/footer.module.js",
    "unpkg": "dist/footer.umd.js",
```
Comment (andyg0808) [inline package.json:14]: I bet you're going to want the same `./` added on the beginning of the rest of these paths, particularly the `index.d.ts` one.

**Review by andyg0808 — APPROVED** [review]

**Comment by andyg0808** [comment]
CR :fish: through 69205ed
deploy_block :+1: in case you want to convert a few more paths to be relative.

**Comment by dhmacs** [comment]
Are we all on the same page with this [Slack discussion](https://inkofpixel.slack.com/archives/C02666ZEY85/p1629838815069000) ? The footer is still using custom made components using styled components.. This will add weight to our apps bundles. I have on my todo list to convert these components to Chakra and embed them into `@ifixit/react-components` (still retaining the composable design, so that customizations can be made without having to publish new versions). Let me know if you still want to publish this and covert the components later, or if you want to wait on my task (I can make the adjustments this week if needed)

**Comment by bhuminson** [comment]
@dhmacs I figured this would be a measure we could take for the time being since it was on your low priority todos, but seeing that it can be done relatively soon, I'll wait on your task and we'll use `@ifixit/react-components` in place of  `@ifixit/footer` in nextjs-community :+1:

**Pull request closed without merging**